app/src/crypto_helper.c

```diff
@@ -18,7 +18,7 @@
 #include "coin.h"
 #include "lisk_base32.h"
 #include "zxmacros.h"
-
+#include "zxmacros_ledger.h"
```

Comment:
zxmacros_ledger should be included automatically after including zxmacros and it needs some SDK specific files that may be conflictive with cpp tests

Reply:
fixed!